h/templates/client/excerpt.html

```diff
@@ -0,0 +1,18 @@
+<div ng-transclude ng-if="!enabled()"></div>
```

Comment:
Should use `controllerAs` in the directive definition and then `vm.isEnabled()` here.